autopush/router/gcm.py

```diff
@@ -115,6 +115,8 @@ def _route(self, notification, uaid_data):
         try:
             gcm = self.gcm[router_data['creds']['senderID']]
             result = gcm.send(payload)
+        except RouterException:
+            raise  # pragma nocover
```

Comment:
Did this somehow get dropped?

Reply:
This may be a merge error.

Reply:
Maybe we should have a test that triggers this to avoid future merge errors?